### PR TITLE
Add mutations for (un)publish multiple pages

### PR DESCRIPTION
**Pull request by akjanik**
Part of #3927 

### Pull Request Checklist

1. [x] Privileged views and APIs are guarded by proper permission checks.
1. [x] All visible strings are translated with proper context.
1. [x] All data-formatting is locale-aware (dates, numbers, and so on).
1. [x] Database queries are optimized and the number of queries is constant.
1. [x] Database migration files are up to date.
1. [x] The changes are tested.
1. [x] The code is documented (docstrings, project documentation).
1. [x] GraphQL schema and type definitions are up to date.
1. [x] Changes are mentioned in the changelog.

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -5,7 +5,7 @@ All notable, unreleased changes to this project will be documented in this file.
 ## [Unreleased]
 - Refactor error handling in mutations #3891 by @maarcingebala @akjanik
 - Use only_fields instead of exclude_fields in gql api - #3940 by @michaljelonek
-
+- Add mutations for (un)publish multiple pages - #3954 by @akjanik
```
Comment (NyanKiyoshi) [inline CHANGELOG.md]: Maybe we could have a better looking sentence in the changelogs?
```suggestion
- Add mutations for publishing and unpublishing multiple pages - #3954 by @akjanik
```

File: tests/conftest.py [hunk 2]
```diff
@@ -752,6 +751,32 @@ def page(db):
...
+        'title': 'Test page',
+        'content': 'test content'}
+    page_1 = Page.objects.create(**data_1)
+    page_2 = Page.objects.create(**data_2)
+    return page_1, page_2
+
+
+@pytest.fixture
+def page_list_unpublished(db):
+    page_1 = Page.objects.create(slug='page-1', title='Page 1',
```
Comment (korycins) [inline tests/conftest.py]: Can you do `create` with `bulk_create`? https://docs.djangoproject.com/en/dev/ref/models/querysets/#bulk-create
Less db queries, the same result.

File: saleor/graphql/core/mutations.py [hunk 3]
```diff
@@ -410,7 +419,7 @@ def perform_mutation(cls, root, info, ids):
                 instance_errors.append(msg)
 
             if not instance_errors:
-                clean_instances.append(instance)
+                clean_instances.append(instance.pk)
```
Comment (NyanKiyoshi) [inline saleor/graphql/core/mutations.py]: I would rename `clean_ids` or something like this.

File: tests/api/test_page.py [hunk 4]
```diff
@@ -210,3 +210,52 @@ def test_paginate_pages(user_api_client, page):
...
+        }
+    """
+    variables = {'ids': [
+        graphene.Node.to_global_id('Page', page.id)
+        for page in page_list]}
+    response = staff_api_client.post_graphql(
+        query, variables, permissions=[permission_manage_pages])
+    content = get_graphql_content(response)
+    for page in page_list:
+        page.refresh_from_db()
```
Comment (korycins) [inline tests/api/test_page.py]: Maybe I am exaggerating but you run refresh 3 times which is equal to 3 db hits. 
You can easily reduce it to one -  `Page.objects.filter(id__in[page_ids])`

File: saleor/graphql/schema.graphql [hunk 5]
```diff
@@ -41,7 +41,7 @@ input AddressInput {
   city: String
   cityArea: String
   postalCode: String
-  country: String!
+  country: String
```
Comment (maarcingebala) [inline saleor/graphql/schema.graphql]: Is this change intentional?

Comment (korycins) [reply saleor/graphql/schema.graphql]: @maarcingebala  I am getting the same on master branch. It seems that some previous PR didnt include rebuilded schema file.

Comment (akjanik) [reply saleor/graphql/schema.graphql]: It seems schema was not generated in #3891
The change is in the initial commit in the PR
https://github.com/mirumee/saleor/pull/3891/commits/31dc1eac3232ce0a86f79096b695bd8f122208b6#diff-556cb55b0af80084e456dbfff8ee36ccL26

Comment (NyanKiyoshi) [reply saleor/graphql/schema.graphql]: For reference this is the diff:
Screenshot 2019-04-11 at 14 14 06

Does look accidental.

Comment (maarcingebala) [reply saleor/graphql/schema.graphql]: I remember that I was reworking the address validation and maybe this change was some part of it. I guess country code doesn't need to be required at the schema level as address validator would check if it's set. Also, maybe it was easier to do the address updates without the country field being required, and that's was the reason for this change.

Comment (korycins) [reply saleor/graphql/schema.graphql]: @maarcingebala  So in that case, we can merge it. Am I right?

Comment (maarcingebala) [reply saleor/graphql/schema.graphql]: I guess yes.

Comment (dominik-zeglen) [reply saleor/graphql/schema.graphql]: Yep
